code/modules/uplink/uplink_items.dm

```diff
@@ -623,7 +623,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/dangerous/sniper
 	name = "Sniper Rifle"
 	desc = "Ranged fury, Syndicate style. Guaranteed to cause shock and awe or your TC back!"
-	item = /obj/item/gun/ballistic/automatic/sniper_rifle/syndicate
+	item = /obj/item/gun/ballistic/sniper_rifle/syndicate
 	cost = 16
```

Comment:
shouldn't the cost of the sniper be lower now that it doesn't come with any ammo?

Reply:
it does come with ammo still, I had the same misunderstanding with how they had worded things lol

The magazines you buy separately from the gun come with no ammo, but the gun itself still comes with a loaded mag.